include/range/v3/range/access.hpp

```diff
@@ -31,11 +31,23 @@
 #include <range/v3/iterator/reverse_iterator.hpp>
 #include <range/v3/iterator/traits.hpp>
 #include <range/v3/utility/static_const.hpp>
+#include <concepts/type_traits.hpp>
 
 #include <range/v3/detail/disable_warnings.hpp>
 
 namespace ranges
 {
+    namespace _safe_
+    {
+        CPP_def
+        (
+            template(typename T)
+            concept Safe,
+                std::is_lvalue_reference<T>::value ||
+                enable_safe_range<concepts::detail::remove_cvref_t<T>>
+        );
+    }
+
```

Comment:
`CPP_def` is no longer a thing.

Reply:
I'm confused, this isn't what the code looks like on my end.

Reply:
... because I didn't push it 🤦‍♂

Reply:
Nit: we don't use PascalCase for concepts any longer.

Reply:
Renamed to lower.